Zend/tests/gh13279.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
```

Comment:
I guess this should go to ext/standard/tests/array

Reply:
Done. TBH the separation of these tests is often quite arbitrary. I usually just throw stuff in Zend.